src/Sylius/Bundle/CurrencyBundle/Resources/config/serializer/Model.Currency.yml

```diff
@@ -0,0 +1,41 @@
+Sylius\Component\Currency\Model\Currency:
+    exclusion_policy: ALL
+    xml_root_name: currency
+    properties:
+        id:
+            expose: true
+            type: integer
+            xml_attribute: true
+            groups: [Default, Detailed]
+        code:
+            expose: true
+            type: string
+            groups: [Default, Detailed]
+        exchangeRate:
+            expose: true
+            type: float
+            groups: [Default, Detailed]
+        enabled:
+            expose: true
+            type: boolean
+            groups: [Default, Detailed]
+        base:
+            expose: true
+            type: boolean
+            groups: [Default, Detailed]
+        createdAt:
+            expose: true
+            type: DateTime
+            groups: [Detailed]
+        updatedAt:
+            expose: true
+            type: DateTime
+            groups: [Detailed]
+    relations:
```

Comment:
I'm stuck with this. 3 tests still fail because the _links collection is not present in JSON response. I have no idea what causes this to be skipped.